src/components/NcRelatedResourcesPanel/NcResource.vue

```diff
@@ -0,0 +1,112 @@
+<!--
+  - @copyright 2022 Christopher Ng <chrng8@gmail.com>
+  -
+  - @author Christopher Ng <chrng8@gmail.com>
+  -
+  - @license AGPL-3.0-or-later
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+  -
+-->
+
+<template>
+	<li class="resource">
+		<div v-tooltip="tooltip" class="resource__desc">
+			<h5>{{ title }}</h5>
+			<p v-if="subtitle">
+				{{ subtitle }}
+			</p>
+		</div>
+		<NcButton :aria-label="labelTranslated"
+			type="tertiary"
+			:href="url"
+			target="_blank">
```

Comment:
👎🏼  this is a user decision and we should not force it.

Reply:
User decision to be able to open the link to the resource?

Reply:
Yes, new tab/window or not is user's decision. Especially since all those links are internal we should not force _blank